### PR TITLE
Even more player funcs

### PR DESCRIPTION
**Pull request by joshlory**

### CODE REVIEW

File: src/main/323C.c [hunk 1]
```diff
@@ -2790,21 +2825,45 @@ s32 func_8002D25C(struct PlayerObj* arg0)
     }
 }
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002D32C);
+s32 func_8002D32C(struct PlayerObj* arg0, s16 arg1, s32 arg2)
+{
+    if (func_8002D41C(arg0, func_8002D724(arg0, D_8013B7F8, arg1), arg2) == 0) {
```
Comment (sozud) [inline src/main/323C.c:2830]: This also matches, either way is fine https://decomp.me/scratch/KA5p0

File: src/main/323C.c [hunk 2]
```diff
@@ -3416,7 +3610,53 @@ INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80035240);
...
+        switch (engine->stage) {
+        case 3:
+            if (engine->unk1E == -1 && engine->checkpoint != 0) {
+                var_a0 = 1;
+            }
+            break;
+        case 5:
+            var_a0 = 2;
+            if (engine->substage != 0) {
+                var_a0 = (engine->checkpoint == 0) * 2;
```
Comment (sozud) [inline src/main/323C.c:3638]: Thinking this was
```
                if(engine_obj_checkpoint == 0)
                {
                    var_a0 = 2;
                } else {
                    var_a0 = 0;
                }
```
https://decomp.me/scratch/n4STP

**Review by sozud — APPROVED** [review]
Great work!

File: src/main/323C.c (hunk 1 above)
Comment (joshlory) [reply src/main/323C.c:2830]: With `./format` this becomes:

```c
if (func_8002D41C(arg0, func_8002D724(arg0, D_8013B7F8, arg1), arg2) == 0 && func_8002D41C(arg0, func_8002D724(arg0, D_8013B7F8 - D_8013B7E0, arg1), arg2) == 0 && func_8002D41C(arg0, func_8002D724(arg0, D_8013B7F8 + D_8013B7E0 - 1, arg1), arg2) == 0) {
```

Comment (sozud) [reply src/main/323C.c:2830]: Ok, either adjusting the .clang-format or keeping the current way is fine with me

Comment (joshlory) [reply src/main/323C.c:2830]: I couldn't figure out how to fix clang-format without setting `ColumnLimit`, going to leave it as-is for now.

**Pull request merged**